app/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
   package="com.criteo.pubsdk_android">
 
   <uses-permission android:name="android.permission.INTERNET"/>
 
+  <!-- MoPub requires minSdk=19, while the SDK had minSdk=16 -->
```

Comment:
Can you add a `See https://developer.android.com/studio/build/manifest-merge` comment please?

Reply:
done